docs/ecctl-getting-started.asciidoc

```diff
@@ -164,14 +168,13 @@ the binary for Elastic Cloud:
 
 [source,yaml]
 ----
-host: https://api.elastic-cloud.com # URL of Elastic Cloud or your ECE installation.
-region: us-east-1 # For ECE installations you can just omit this setting.
+host: https://api.elastic-cloud.com # URL of your ECE installation.
```

Comment:
Worth calling out `URL of your ECE API endpoint` specifically. ECE UI will have a different endpoint.